### PR TITLE
Added ability to cancel entering chat with home button #35

### PR DESCRIPTION
**Pull request by rutulganatra**

### CODE REVIEW

File: src/routes/Chat/components/Modals/IntroModal.jsx [hunk 1]
```diff
@@ -104,15 +116,21 @@ const IntroModal = ({open, setUser, setWsApi, setUserList}) => {
...
-                        variant={'contained'}
-                        color={'success'}
-                        type={'submit'}
-                        loading={isLoading}
-                        disabled={inputValue.length<3}
-                    >
-                        Enter
-                    </LoadingButton>
+                    <Stack spacing={{xs: 1, sm: 2}} direction="row" useFlexGap>
+                        <Button variant="contained" color='inherit' onClick={handleClick}>
```
Comment (readyyyk) [inline src/routes/Chat/components/Modals/IntroModal.jsx:120]: use[`icon-button`](https://mui.com/material-ui/api/icon-button/) instead of plain button

Comment (rutulganatra) [reply src/routes/Chat/components/Modals/IntroModal.jsx]: Can you please review it now?

**Review by readyyyk — APPROVED** [review]

**Comment by readyyyk** [comment]
run `npm run lint` before commit

**Comment by rutulganatra** [comment]
I did npm run lint and made some changes for it and pushed it. Please review it once

**Pull request merged**